src/kernel/pgtbl.c

```diff
@@ -1,187 +1,234 @@
+/******************************************************************************
+Filename    : pgtbl.c
+Author      : Gabriel Parmer
+Date        : ?/?/2015
+Licence     : GPL v2; see COPYING for details.
+Description : The page table operations that does not need special optimizations.
+******************************************************************************/
```

Comment:
Comment style is completely off from what's used in composite right now!

Reply:
This is the new comment style I want to use. This a block comment style.